testStrategies/commonToAllTestStrategies.go

```diff
@@ -50,23 +50,23 @@ type ResponseValue struct {
 
 // TestDefinition encapsulates the XML data.
 type TestDefinition struct {
-	XMLName             xml.Name             `xml:"testDefinition" json:"testDefinition"`
-	TestName            string               `xml:"testName" json:"testName"`
-	OverrideHost        string               `xml:"overrideHost" json:"overrideHost"`
-	OverridePort        string               `xml:"overridePort" json:"overridePort"`
-	HTTPMethod          string               `xml:"httpMethod" json:"httpMethod"`
-	Description         string               `xml:"description" json:"description"`
-	BaseURI             string               `xml:"baseUri" json:"baseUri"`
-	Multipart           bool                 `xml:"multipart" json:"multipart"`
-	Payload             string               `xml:"payload" json:"payload"`
-	MultipartPayload    []multipartFormField `xml:"multipartPayload>multipartFormField" json:"multipartPayload"`
-	ResponseStatusCode  int                  `xml:"responseStatusCode" json:"responseStatusCode"`
-	ResponseContentType string               `xml:"responseContentType" json:"responseContentType"`
-	Headers             []Header             `xml:"headers>header" json:"headers"`
-	ResponseValues      []ResponseValue      `xml:"responseProperties>value" json:"responseValues"`
-	PreThinkTime        int64
-	PostThinkTime       int64
-	ExecWeight          string
+	XMLName             xml.Name             `xml:"testDefinition"`
+	TestName            string               `xml:"testName"`
+	OverrideHost        string               `xml:"overrideHost"`
+	OverridePort        string               `xml:"overridePort"`
+	HTTPMethod          string               `xml:"httpMethod"`
+	Description         string               `xml:"description"`
+	BaseURI             string               `xml:"baseUri"`
+	Multipart           bool                 `xml:"multipart"`
+	Payload             string               `xml:"payload"`
+	MultipartPayload    []multipartFormField `xml:"multipartPayload>multipartFormField"`
+	ResponseStatusCode  int                  `xml:"responseStatusCode"`
+	ResponseContentType string               `xml:"responseContentType"`
+	Headers             []Header             `xml:"headers>header"`
+	ResponseValues      []ResponseValue      `xml:"responseProperties>value"`
+	PreThinkTime        int64                `xml:"preThinkTime"`
```

Comment:
Don't include this change (prethinktime, postthinktime, execweight).

@ConorSmyth Should this be merging to develop?